### PR TITLE
[PREVIEW] - remove start page and navigate directly to idam

### PR DESCRIPTION
**Pull request by dtaylor7**
# Description

[DIV-3576 - Idam login should be first page as part of DN journey](https://tools.hmcts.net/jira/browse/DIV-3576)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,5 +21,15 @@ Locally this will also create a new URL of ```"/session"``` where you can view a
...
+####Run Functional tests locally
+
+1. create local.yml file in config folder with the contents:
+```
+tests:
+  functional:
+    proxy:
+    proxyByPass:
+```
+2. run `yarn test:functional`
```
Comment (hosainnet) [inline README.md]: Nice, just been playing with this 👍

File: test/functional/smoke/index.js [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
-  I.seeCurrentUrlEquals(StartPage.path);
-  I.see(StartPageContent.en.title);
-});
+
+  const currentPath = await I.getCurrentUrl();
+
+  if (currentPath === IdamMockLogin.path) {
+    I.seeCurrentUrlEquals(IdamMockLogin.path);
+  } else {
+    I.seeInCurrentUrl('/login?');
```
Comment (hosainnet) [inline test/functional/smoke/index.js]: Are we able to use the `I.testIdamPage` here instead?

Comment (dtaylor7) [reply test/functional/smoke/index.js]: i think its enough to see the page has loaded?

File: README.md (hunk 1 above)
Comment (dtaylor7) [reply README.md]: 👍

File: test/functional/smoke/index.js (hunk 2 above)
Comment (hosainnet) [reply test/functional/smoke/index.js]: 👍 
Another option is `I.seeInCurrentUrl('login')` as both mock and the real page have a `login` in the URL but this works too. Not a big deal anyway

Comment (dtaylor7) [reply test/functional/smoke/index.js]: yep could do... although do i wanna way another 20mins...

**Pull request merged**